nikola/conf.py.in

```diff
@@ -208,6 +212,57 @@ COMPILERS = ${COMPILERS}
 # Warning: this option will change its default value to False in v8!
 WRITE_TAG_CLOUD = True
 
+# Generate pages for categories. The site must have at least two categories
+# for this option to take effect. It wouldn't build for just one category.
+POSTS_CATEGORIES = True
+
+# Setting this to False generates a list page instead of an index. Indexes
+# are the default and will apply GENERATE_ATOM if set.
+# POSTS_CATEGORY_ARE_INDEXES = True
+
+# Each post and category page will have an associated color that can be used
+# to style them with a recognizable color detail across your site. A color
+# is assigned to  each category based on shifting the hue of your THEME_COLOR
+# at least 7.5 % while leaving the lightness and saturation untouched in the
+# HUSL colorspace. You can overwrite colors by assigning them colors in HEX.
+POSTS_CATEGORY_COLORS = {
```

Comment:
should be commented out by default, along with POST_CATEGORY_DESCRIPTIONS and POST_CATEGORY_TITLE